flixel/util/FlxCollision.hx

```diff
@@ -21,15 +24,17 @@ import flixel.tile.FlxTileblock;
 class FlxCollision 
 {
 	// Optimization: Local static vars to reduce allocations
-	private static var pointA:Point = new Point();
```

Comment:
That's interesting, I didn't know Flixel point / rects were faster than their OpenFL counterparts.